Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -29,7 +30,72 @@ extension BrowserViewController {
         }
     }
     
-    func destinationMenuSection(_ menuController: MenuViewController) -> some View {
+    func privacyFeaturesMenuSection(_ menuController: MenuViewController) -> some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Spacer(minLength: 15.0)
+            
+            Text(verbatim: Strings.PrivacyFeature.menuSectionTitle.uppercased())
+                .font(.callout)
+                .fontWeight(.semibold)
+                .lineLimit(1)
+                .foregroundColor(Color(.braveLabel)
+            ).padding(.horizontal, 14)
```

Comment:
closing parenthesis should be on previous line

Reply:
Fix at https://github.com/brave/brave-ios/pull/4424/commits/67ce7646e8e57fc92c7941b2ddbdf30450aaadec